v0/scripts/PublishPrices.js

```diff
@@ -12,7 +12,11 @@ const alphaVantageKey = process.env.ALPHAVANTAGE_API_KEY;
 const barchartKey = process.env.BARCHART_API_KEY;
 
 function stripApiKey(str, key) {
-  return str.replace(key, "{redacted}");
+  if (key) {
+    return str.replace(key, "{redacted}");
```

Comment:
Its possible for `key` to be an empty string. Probably a cleaner way to address this, but this seemed the easiest for now.

Reply:
thanks for the fix